templates/bot/csharp/command-and-response/.gitignore

```diff
@@ -1,9 +1,3 @@
-# User-specific files
-*.suo
-*.user
```

Comment:
Keep `.user`? It's possible to have a `xxx.csproj.user` file and that's project-level.